compiler/src/test/java/com/squareup/anvil/compiler/dagger/AssistedInjectGeneratorTest.kt

```diff
@@ -642,12 +674,14 @@ public final class AssistedService_Factory {
 
   private fun compile(
     @Language("kotlin") vararg sources: String,
-    block: JvmCompilationResult.() -> Unit = { }
+    previousCompilationResult: JvmCompilationResult? = null,
+    block: JvmCompilationResult.() -> Unit = { },
```

Comment:
- [ ] Undo - previously made this while trying out other test cases but this proved to not be needed.